website/static/css/pages/wiki-page.css

```diff
@@ -11,6 +11,15 @@
     padding: 10px;
 }
 
+.osf-panel {
```

Comment:
@haoyuchen1992 This seems to be a more general rule that might be used outside of the wiki page; should this go in style.css?